api/envoy/api/v2/endpoint/load_report.proto

```diff
@@ -105,6 +106,49 @@ message EndpointLoadMetricStats {
   double total_metric_value = 3;
 }
 
+// Stats per downstream host.
+message DownstreamEndpointStats {
+  // Downstream host address.
+  core.Address address = 1;
+
+  // Metadata passed directly to the management server.
+  google.protobuf.Struct metadata = 5;
+
+  // The total number of requests per downstream host
+  // successfully completed by the endpoint. A single HTTP or gRPC
+  // request or stream is counted as one request. A TCP connection is
+  // also treated as one request. There is no explicit total_requests
+  // field below for an endpoint, but it may be inferred from:
+  //
+  // .. code-block:: none
+  //
+  //   total_requests = total_successful_requests + total_requests_in_progress +
+  //     total_error_requests
+  //
+  // The total number of requests per downstream host successfully
+  // completed by the endpoints in the locality. These include non-5xx
+  // responses for HTTP, where errors originate at the client and the
+  // endpoint responded successfully. For gRPC, the grpc-status values
+  // are those not covered by total_error_requests below.
+  uint64 total_successful_requests = 2;
+
+  // The total number of unfinished requests per downstream host for
+  // this endpoint.
+  uint64 total_requests_in_progress = 3;
+
+  // The total number of requests per downstream host that failed
+  // due to errors at the endpoint. For HTTP these are responses with
+  // 5xx status codes and for gRPC the grpc-status values:
+  //
+  //   - DeadlineExceeded
+  //   - Unimplemented
+  //   - Internal
+  //   - Unavailable
+  //   - Unknown
+  //   - DataLoss
+  uint64 total_error_requests = 4;
```

Comment:
We cant get this information as it is only present at the client and not backend.

We need to talk about why this load report is required, somewhere

Reply:
We can probably get at it as we sit on the return path for any request. 

Definitely need to motivate this better. I get why it matters for gRPC-LB use cases, are there middle proxy use cases that this might also be applicable to? I'm sold on the usefulness of this for your use case, but I'd like to ensure we communicate where this fits into the LRS/EDS grand scheme of things to other users.